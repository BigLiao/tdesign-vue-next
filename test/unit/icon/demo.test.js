// import { mount } from '@vue/test-utils';

// todo: waiting for test bugs fixed
// unit test for component in examples.
describe('Icon', () => {
  it('base demo works fine', () => {
    // const demo = require('@/examples/icon/demos/base.vue');
    // const wrapper = mount(demo);
    // expect(wrapper.element).toMatchSnapshot();
    expect(true).toBe(true);
  });
});
