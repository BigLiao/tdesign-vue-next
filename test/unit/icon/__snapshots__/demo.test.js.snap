// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon base demo works fine 1`] = `
<div
  class="t-demo-iconfont"
>
  <div
    class="t-demo-row"
  >
    <svg
      class="t-icon t-icon-loading"
      style="color: rgb(153, 153, 153);"
    >
      <use
        href="#t-icon-loading"
      />
    </svg>
    <svg
      class="t-icon t-icon-close"
    >
      <use
        href="#t-icon-close"
      />
    </svg>
    <svg
      class="t-icon t-icon-check-circle-filled"
    >
      <use
        href="#t-icon-check-circle-filled"
      />
    </svg>
  </div>
  <br />
  <div
    class="t-demo-row"
  >
    <svg
      class="t-icon t-icon-check-circle-filled t-size-s"
    >
      <use
        href="#t-icon-check-circle-filled"
      />
    </svg>
    <svg
      class="t-icon t-icon-check-circle-filled"
    >
      <use
        href="#t-icon-check-circle-filled"
      />
    </svg>
    <svg
      class="t-icon t-icon-check-circle-filled t-size-m"
    >
      <use
        href="#t-icon-check-circle-filled"
      />
    </svg>
    <svg
      class="t-icon t-icon-check-circle-filled t-size-l"
    >
      <use
        href="#t-icon-check-circle-filled"
      />
    </svg>
    <svg
      class="t-icon t-icon-check-circle-filled"
      style="font-size: 25px;"
    >
      <use
        href="#t-icon-check-circle-filled"
      />
    </svg>
    <svg
      class="t-icon t-icon-check-circle-filled"
      style="font-size: 2em;"
    >
      <use
        href="#t-icon-check-circle-filled"
      />
    </svg>
  </div>
  <br />
  <div
    class="t-demo-row"
  >
    <svg
      class="t-icon t-icon-check-circle-filled"
      style="color: red;"
    >
      <use
        href="#t-icon-check-circle-filled"
      />
    </svg>
    <svg
      class="t-icon t-icon-check-circle-filled"
      style="color: green;"
    >
      <use
        href="#t-icon-check-circle-filled"
      />
    </svg>
    <svg
      class="t-icon t-icon-check-circle-filled"
      style="color: orange;"
    >
      <use
        href="#t-icon-check-circle-filled"
      />
    </svg>
  </div>
</div>
`;
